test/unit/comment_test.rb

```diff
@@ -136,28 +136,28 @@ class DrupalCommentTest < ActiveSupport::TestCase
   test "should relate answer comments to user and answer but not node" do
     answer = answers(:one)
     user = users(:bob)
-    comment = DrupalComment.new(comment: 'Test comment')
+    comment = Comment.new(comment: 'Test comment')
     comment.drupal_users = user
     comment.answer = answer
 
     assert comment.save
-    assert_equal user.drupal_comments.last, comment
-    assert_equal answer.drupal_comments.last, comment
-    assert_not_equal answer.node.drupal_comments.last, comment
+    assert_equal user.comments.last, comment
```

Comment:
I think this isn't an issue now because there's only one comment for the user `bob`, but it will be if the fixtures are ever expanded. Best make this `.first` or add the ordering to make it extra clear to future contributors.